README.md

```diff
@@ -1 +1,915 @@
+```
+
+```text
+Flags:
+      --force   if true, delete resources and ignore errors
+  -h, --help    help for delete
+      --prune   if true, delete a namespace with a project
+```
+
+### The `list` command
+
+List all deployments.
+
+```text
+$ rasactl list
+CURRENT	NAME         	STATUS 	RASA PRODUCTION	RASA WORKER	ENTERPRISE	VERSION
+       	hopeful-haibt	Running	2.8.1          	2.8.1      	inactive  	0.42.0
+*      	vibrant-yalow	Running	2.8.1          	2.8.1      	inactive  	0.42.0
+```
+
+The `*` in the `CURRENT` field indicates a deployment that is used as default. It means that every time when you execute `rasactl` command without defining the deployment name, the deployment marked with `*` is used.
+
+A deployment is marked as `CURRENT` if:
+
+- there is a `.rasactl` file that includes a deployment name in your current working directory. The file is automatically created if you run the `rasactl start` command with the `--project` or `--project-path` flag
+- there is only one deployment
+- you set the current deployment by using the `rasactl config use-deployment` command
+
+### The `status` command
+
+Show the status of a deployment.
+
+```text
+Usage:
+  rasactl status [DEPLOYMENT-NAME] [flags]
+```
+
+```text
+Examples:
+  # Show status for the 'example' deployment.
+  $ rasactl status example
+
+  # Show status for the 'example' deployment along with details.
+  $ rasactl status example --details
+```
+
+```text
+Flags:
+  -d, --details   show detailed information, such as running pods, helm chart status
+  -h, --help      help for status
+```
+
+Example output:
+
+```text
+$ rasactl status vibrant-yalow
+Name:                   	vibrant-yalow
+Status:                 	Running
+URL:                    	http://vibrant-yalow.rasactl.localhost
+Version:                	0.42.0
+Enterprise:             	inactive
+Rasa production version:	2.8.1
+Rasa worker version:    	2.8.1
+Project path:           	/home/ubuntu/test
+```
+
+### The `config use-deployment` command
+
+Sets the current-deployment in the configuration file.
+
+If you have multiple Rasa X deployments, and you are not in a project directory you have to explicitly define the deployment name during command execution.
+You can define a deployment that is used as a current one by using the `rasa config use-deployment` command.
+
+```text
+Usage:
+  rasactl config use-deployment DEPLOYMENT-NAME [flags]
+```
+
+```text
+Examples:
+  # Set the 'example' deployment as the current deployment.
+  $ rasactl config use-deployment example
+```
+
+```text
+Flags:
+  -h, --help   help for use-deployment
+```
+
+### The `connect rasa` command
+
+Connect Rasa OSS (Open Source Server) to Rasa X deployment.
+
+The command prepares a configuration that's required to connect Rasa X deployment and run a local Rasa server.
+
+It's required to have the 'rasa' command accessible by rasactl.
+
+The command works only if Rasa X deployment runs on a local Kubernetes cluster runs on kind.
```

Comment:
should be "runs on kind", right? or "on a local Kubernetes cluster managed with `kind`"?

Reply:
The latter, technically `kind` only set up the cluster, it doesn't do anything extra.

Reply:
ok! I think "runs on a local Kubernetes cluster runs on kind" looked like a typo to me so i was a bit confused 😄

Reply:
I've updated it.